site/content/docs/0.0/examples/fake/index.html

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
**question(non-blocking):** Just for my knowledge, what is this file used for ? I don't see any "Fake" on the exampes page of the netlify published doc.

Reply:
In Boosted, you can [package and download the examples](https://boosted.orange.com/docs/5.3/examples/). The script in our `package.json` that handles this expects files in the examples directory, including at least one CSS file. To keep things simple, I created a temporary hidden example to avoid modifying the build/packaging process. This example is intentionally not displayed on the examples page.

Consider this a temporary hack, knowing that we will have at least the "Masonry" example in the future, so this fake example will be removed soon.

Reply:
ok thanks, I understand.